os/tools/compression/Makefile

```diff
@@ -90,29 +90,25 @@ $(APPNAME): Makefile $(CONFIG) $(CCONFIG) $(COMPHEADER) $(OBJECTS)
 	@$(CC) $(LDFLAGS) $(OBJECTS) $(LIBFILES) -o $@
```

Comment:
Is this CC? not LD?

Reply:
It is CC , we are building the tool for compressing image for ubuntu machine. Not cross compiling